src/app/api/translate/route.ts

```diff
@@ -0,0 +1,36 @@
+import { NextResponse } from "next/server";
+import Client from "groq-sdk";
+const client = new Client({
+  apiKey: process.env.GROQ_API_KEY || "",
```

Comment:
**Settings: use a settings.ts file to manage ENV files**


https://github.com/nmaswood/fgpt/blob/main/typescript/packages/api/src/settings.ts